### PR TITLE
Add support of Instant Shopping API

### PR DESCRIPTION
**Pull request by alexions**

### CODE REVIEW

**Review by freethan — CHANGES_REQUESTED** [review]
Please see https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file so as to ignore parts of the file that might not conform to the standard, e.g. long @phpdoc tags.

This way we can restore useful documentation

**Review by freethan — APPROVED** [review]

File: src/Klarna/Rest/InstantShopping/ButtonKeys.php [hunk 1]
```diff
@@ -92,12 +92,17 @@ public function create(array $data)
      * @throws ConnectorException When the API replies with an error response
      * @throws RequestException   When an error is encountered
      * @throws \RuntimeException  If the API replies with an unexpected response
+     * @throws \RuntimeException         If key was not specified when creating a resource
```
Comment (freethan) [inline src/Klarna/Rest/InstantShopping/ButtonKeys.php]: Indentation :/

File: src/Klarna/Rest/InstantShopping/ButtonKeys.php [hunk 2]
```diff
@@ -73,10 +74,12 @@ public function create(array $data)
     {
         $data = $this->post(self::$path, $data)
             ->status('201')
-            ->contentType('application/json')
-            ->getJson();
+            ->contentType('application/json');
```
Comment (freethan) [inline src/Klarna/Rest/InstantShopping/ButtonKeys.php:77]: Please fix this variable ($data), I prefer the solution in line 99 where we don't overwrite it as it comes in as an argument.

File: src/Klarna/Rest/InstantShopping/ButtonKeys.php [hunk 3]
```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Copyright 2014 Klarna AB
```
Comment (freethan) [inline src/Klarna/Rest/InstantShopping/ButtonKeys.php]: Copyright 2019 :)

File: src/Klarna/Rest/InstantShopping/Orders.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Copyright 2014 Klarna AB
```
Comment (freethan) [inline src/Klarna/Rest/InstantShopping/Orders.php]: Copyright 2019 :)

**Comment by alexions** [comment]
Fixed all the comments

**Pull request merged**